lib/generators/sufia/templates/workflow.json.erb

```diff
@@ -9,19 +9,45 @@
                     "name": "deposit",
                     "from_states": [],
```

Comment:
Do we need to add a `from_state` here for `changes_requested`? 

And, if we do, will that break the ability to execute this action when not currently in a state, since new deposits aren't yet in a state? That also implies we'll need to add a role here to match the role in the final action declared at the bottom of this file, which implies that we need code that automatically adds a depositing role to all registered users, else they won't be able to participate in this workflow.

Tagging @jcoyne and @jeremyf too for feedback on the above.